src/python/pants/option/options_test.py

```diff
@@ -853,171 +869,180 @@ def test_enum_option_type_parse_error(self):
   def assertOptionWarning(self, w, option_string):
     single_warning = assert_single_element(w)
     self.assertEqual(single_warning.category, DeprecationWarning)
-    warning_message = single_warning.message
-    self.assertIn("will be removed in version", str(warning_message))
-    self.assertIn(option_string, str(warning_message))
-
-  def test_deprecated_options_flag(self):
```

Comment:
Originally, there were two distinct tests for `deprecated_options`. I consolidated them into one and used a new inlined helper function to reduce duplication